uniffi_bindgen/src/bindings/kotlin/mod.rs

```diff
@@ -73,11 +74,22 @@ impl BindingGenerator for KotlinBindingGenerator {
             kt_file.push(format!("{}.kt", ci.namespace()));
             fs::write(&kt_file, generate_bindings(config, ci)?)?;
             if settings.try_format_code {
-                if let Err(e) = Command::new("ktlint").arg("-F").arg(&kt_file).output() {
-                    println!(
-                        "Warning: Unable to auto-format {} using ktlint: {e:?}",
-                        kt_file.file_name().unwrap(),
-                    );
+                // #2247 reports this sometimes taking *minutes*
+                let start = Instant::now();
+                match Command::new("ktlint").arg("-F").arg(&kt_file).output() {
+                    Ok(_) => {
+                        println!(
+                            "ktlint took {}s - {}",
+                            start.elapsed().as_secs(),
```

Comment:
If it takes longer than say 5 seconds you should suggest how to disable it.

Reply:
I like this idea, but I don't think that message should be tied to a any specific time.  Maybe there could be a printout that looks something like `running ktlint (use --no-format to disable this)`.  In general, I think we should be pushing users to specify that flag.

Reply:
I guess we might as well just print your suggested message and not bother printing how long it took, which actually was the original idea in the linked issue - if someone is watching it because it is taking longer than expected, that's still all they need to determine what the problem is.